lib/generators/boring/webmock/install/install_generator.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Boring
+  module Webmock
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds webmock gem to the application"
+
+      SUPPORTED_TEST_FRAMEWORKS = %w[rspec minitest]
+
+      class_option :test_gem, type: :string,
```

Comment:
Also, I think it is fine if we default the framework to minitest if no option is present.

Reply:
We are not able to use "test_framework" option which would be a good naming for this because it's being used by Rails::Generator::Base. See: https://github.com/search?q=repo%3Arails%2Frails+path%3A%2F%5Erailties%5C%2Flib%5C%2Frails%5C%2Fgenerators%5C%2F%2F+test_framework&type=code 

I thought it would be better not to have any conflict with the base class so prefixing with "app_" here. Please let me know if we still have to use test_framework, I will research for ways to override base generator in that case.

I have added the comment to the top of this option so others know about the decision as well.